pfcp/adapter/adapter.go

```diff
@@ -68,6 +68,26 @@ func HandleAdapterPfcpRsp(pfcpMsg message.Message, evtData *udp.PfcpEventData) e
 	return nil
 }
 
+func FindUEIPAddress(createdPDRIEs []*ie.IE) net.IP {
```

Comment:
Placeholder comment to understand why this function is needed if UE IP address is assigned by SMF.

Reply:
The UE IP addressed can either be assigned by the SMF or the UPF depending on how the core is configured. The upf has a `enable_ue_ip_alloc` feature flag that can be used